code/modules/antagonists/changeling/powers/shriek.dm

```diff
@@ -43,16 +43,23 @@
 	chemical_cost = 20
 	dna_cost = 1
 	disabled_by_fire = FALSE
+	COOLDOWN_DECLARE(dissonant_shriek_cooldown) //BUBBER EDIT: DECLARES A COOLDOWN
 
 /datum/action/changeling/dissonant_shriek/sting_action(mob/user)
 	..()
 	if(user.movement_type & VENTCRAWLING)
 		user.balloon_alert(user, "can't shriek in pipes!")
 		return FALSE
+	//BUBBER EDIT: NO PULSE IF YOU'RE ON COOLDOWN
+	if(!COOLDOWN_FINISHED(src, dissonant_shriek_cooldown))
+		user.balloon_alert(user, "Your throat is sore!")
```

Comment:
balloon alert standard usually doesnt include capitalization, and usually not "your" instead just opting to say "throat is sore!"